internal/podman/client.go

```diff
@@ -144,8 +144,11 @@ type Client interface {
 	CreateVolume(ctx context.Context, name string) ([]byte, error)
 	RunContainer(ctx context.Context, opts RunContainerOpts) ([]byte, error)
 	CopyFileToContainer(ctx context.Context, localFile string, containerName string, filePathInContainer string) ([]byte, error)
-	StopContainers(ctx context.Context, names ...string) ([]byte, error)
-	RemoveContainers(ctx context.Context, names ...string) ([]byte, error)
+	StopContainers(ctx context.Context, ids ...string) ([]byte, error)
+	StartContainers(ctx context.Context, ids ...string) ([]byte, error)
+	UnpauseContainers(ctx context.Context, ids ...string) ([]byte, error)
+	StopMongoD(ctx context.Context, id string) error
+	RemoveContainers(ctx context.Context, ids ...string) ([]byte, error)
```

Comment:
unless required by any of the podman commands, I would prefer to keep this interface consistent and use names instead of IDs:
```suggestion
	StopContainers(ctx context.Context, names ...string) ([]byte, error)
	StartContainers(ctx context.Context, names ...string) ([]byte, error)
	UnpauseContainers(ctx context.Context, names ...string) ([]byte, error)
	StopMongoD(ctx context.Context, names string) error
	RemoveContainers(ctx context.Context, names ...string) ([]byte, error)
```